docs/src/_layouts/homepage.pug

```diff
@@ -267,7 +267,7 @@ block content
 
     .text-center.mt-2
       .text-1.mb-3.leading-2
-        span.block.mb-1.sm_inline.whitespace-no-wrap DocSearch 2015-2018
+        span.block.mb-1.sm_inline.whitespace-no-wrap DocSearch 2015-2019
```

Comment:
let's maybe just leave out the end year, what do you think @s-pace 
```suggestion
        span.block.mb-1.sm_inline.whitespace-no-wrap DocSearch 2015-now
```

Reply:
Yes indeed, please commit this suggestion @svenluijten, I do not want to close your PR :)